src/home/components/home-banner/HomeBanner.tsx

```diff
@@ -0,0 +1,60 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router';
+import { getConfig } from '@edx/frontend-platform';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { Form, useToggle, SearchField } from '@openedx/paragon';
+
+import { ROUTES } from '@src/routes';
+import HomeOverlayHtmlSlot from '@src/plugin-slots/HomeOverlayHtmlSlot/HomeOverlayHtmlSlot';
+import HomePromoVideoButtonSlot from '@src/plugin-slots/HomePromoVideoButtonSlot/HomePromoVideoButtonSlot';
+import HomePromoVideoModalSlot from '@src/plugin-slots/HomePromoVideoModalSlot/HomePromoVideoModalSlot';
+
+import messages from './messages';
+
+const HomeBanner = () => {
+  const intl = useIntl();
+  const navigate = useNavigate();
+  const [searchValue, setSearchValue] = useState('');
+  const [isOpen, open, close] = useToggle(false);
+
+  const handleSearch = () => navigate(`${ROUTES.COURSES}?search_query=${searchValue}`);
+
+  const searchField = getConfig().ENABLE_COURSE_DISCOVERY && (
+    <Form.Group className="w-100 mb-0 mt-4.5">
```

Comment:
In my testing this looked fine with just

```suggestion
    <Form.Group className="mt-4.5">
```

Removing `mb-0` does add a very small margin at the bottom, but it looked fine to me and didn't feel out of place. `w-100` just isn't needed anymore because this is in a `Container`.